scripts/tcpdirect_misc/tcpdirect.spec

```diff
@@ -8,13 +8,16 @@
 #
 # To build a binary RPM from a source RPM:
 #   rpmbuild --define "onload_tarball <onload_tarball>" --rebuild ~/rpmbuild/SRPMS/tcpdirect-<version>-1.src.rpm
+#
+# If you want debug binary packages add:
+#   --define "debug true"
```

Comment:
I don't think debug packages should be an rpm build option - these are highly undesirable.

Better would be for the debug versions either to exist in parallel within the package, or, event better, in a debug package. (Not to be confused with the debuginfo and debugsource packages which are different again and we should also make sure we are not inhibiting their automatic creation.)